src/fund.jl

```diff
@@ -30,211 +34,229 @@ include("ImpactSeaLevelRiseComponent.jl")
 include("ImpactAggregationComponent.jl")
 include("VslVmorbComponent.jl")
 
-function constructfund(;nsteps=1050)
-    m = Model()
+export 
+    FUND,       # Default version of fund
+    getfund     # Function that returns a version of fund allowing for different user specifications
 
-    setindex(m, :time, collect(1950:1950+nsteps))
-    setindex(m, :regions, ["USA", "CAN", "WEU", "JPK", "ANZ", "EEU", "FSU", "MDE", "CAM", "LAM", "SAS", "SEA", "CHI", "MAF", "SSA", "SIS"])
+
+const global default_nsteps = 1050
+const global default_datadir = joinpath(dirname(@__FILE__), "..", "data")
+const global default_params = nothing
+  
+function getfund(; nsteps = default_nsteps, datadir = default_datadir, params = default_params)
 
     # ---------------------------------------------
-    # Create components
+    # Load parameters
     # ---------------------------------------------
-    addcomponent(m, scenariouncertainty, :scenariouncertainty)
-    addcomponent(m, population, :population)
-    addcomponent(m, geography, :geography)
-    addcomponent(m, socioeconomic, :socioeconomic)
-    addcomponent(m, emissions, :emissions)
-    addcomponent(m, climateco2cycle, :climateco2cycle)
-    addcomponent(m, climatech4cycle, :climatech4cycle)
-    addcomponent(m, climaten2ocycle, :climaten2ocycle)
-    addcomponent(m, climatesf6cycle, :climatesf6cycle)
-    addcomponent(m, climateforcing, :climateforcing)
-    addcomponent(m, climatedynamics, :climatedynamics)
-    addcomponent(m, biodiversity, :biodiversity)
-    addcomponent(m, climateregional, :climateregional)
-    addcomponent(m, ocean, :ocean)
-    addcomponent(m, impactagriculture, :impactagriculture)
-    addcomponent(m, impactbiodiversity, :impactbiodiversity)
-    addcomponent(m, impactcardiovascularrespiratory, :impactcardiovascularrespiratory)
-    addcomponent(m, impactcooling, :impactcooling)
-    addcomponent(m, impactdiarrhoea, :impactdiarrhoea)
-    addcomponent(m, impactextratropicalstorms, :impactextratropicalstorms)
-    addcomponent(m, impactforests, :impactforests)
-    addcomponent(m, impactheating, :impactheating)
-    addcomponent(m, impactvectorbornediseases, :impactvectorbornediseases)
-    addcomponent(m, impacttropicalstorms, :impacttropicalstorms)
-    addcomponent(m, vslvmorb, :vslvmorb)
-    addcomponent(m, impactdeathmorbidity, :impactdeathmorbidity)
-    addcomponent(m, impactwaterresources, :impactwaterresources)
-    addcomponent(m, impactsealevelrise, :impactsealevelrise)
-    addcomponent(m, impactaggregation, :impactaggregation)
+
+    if params == nothing
+        parameters = loadparameters(datadir)
+    else
+        parameters = params
+    end
 
     # ---------------------------------------------
-    # Connect parameters to variables
+    # Create model
     # ---------------------------------------------
 
-    connectparameter(m, :geography, :landloss, :impactsealevelrise, :landloss)
+    FUND = Model()
```

Comment:
Can we use ``m`` as the var name here? That seems to be the pattern we've used across other repos.